packages/grid/x-data-grid/src/utils/exportAs.ts

```diff
@@ -10,9 +10,10 @@ import { GridExportFormat } from '../models/gridExport';
  * https://unpkg.com/browse/@progress/kendo-file-saver@1.0.7/dist/es/save-as.js
  * https://github.com/ag-grid/ag-grid/blob/9565c219b6210aa85fa833c929d0728f9d163a91/community-modules/csv-export/src/csvExport/downloader.ts
  */
-export function exportAs(
+
+export function exportAs<ExtraExtensions extends string>(
   blob: Blob,
-  extension: GridExportFormat = 'csv',
+  extension: GridExportExtension | ExtraExtensions = 'csv',
   filename: string = document.title,
```

Comment:
A small detail but the page may not have title, for instance, the playground doesn't have. In this case it download as "extension.extension".

```suggestion
  filename: string = document.title || 'untitled',
```